fe/fe-core/src/main/java/com/starrocks/planner/PlanNode.java

```diff
@@ -111,6 +111,8 @@ abstract public class PlanNode extends TreeNode<PlanNode> {
 
     protected List<Expr> conjuncts = Lists.newArrayList();
 
+    protected List<Expr> precedingFilterConjuncts = Lists.newArrayList();
```

Comment:
if it's only used in `XXXScanNode`, it's better to put it into the `ScanNode` rather than `PlanNode`